orchagent/portsorch.cpp

```diff
@@ -4228,6 +4264,13 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 /* create host_tx_ready field in state-db */
                 initHostTxReadyState(p);
 
+                // Restore admin status if the port was brought down
+                if (admin_status != p.m_admin_state_up)
```

Comment:
@stepanblyschak why special handling for admin_status? Isn't it already taken care here https://github.com/sonic-net/sonic-swss/blob/9c995f0c83892ae9a4f6657c1c4673164e803169/orchagent/port/porthlpr.cpp#L736

Reply:
@prgeor Setting some attributes require bringing port admin down - like speed. Hence, at the end of an update we need to bring port admin up. If we'd aggregate all fvs on update, then we don't need any special handling, admin_status is always set as per APPL_DB.
However, because this aggregation is removed we need to take a special care to restore port admin state.